tsconfig.json

```diff
@@ -1,17 +1,37 @@
 {
   "compilerOptions": {
-    "target": "es2019",
-    "module": "esnext",
-    "moduleResolution": "Node",
-    "strict": true,
-    "jsx": "preserve",
-    "importHelpers": true,
-    "allowJs": true,
-    "skipLibCheck": true,
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "target": "es2021",
+    "verbatimModuleSyntax": true,
+
+    "allowJs": true,
     "resolveJsonModule": true,
-    "sourceMap": true,
+    "moduleDetection": "force",
+
+    "strict": true,
+    "noUncheckedIndexedAccess": true,
+
+    "moduleResolution": "Node",
+    "module": "ESNext",
+    "noEmit": true,
+
+    "importHelpers": true,
+    "lib": [
+      "es2021",
```

Comment:
According to the comment [here](https://github.com/valantic/vue-template/pull/346/files/39fa63ca8679c17fe2a6f385401ce6a0465e5ead..611478488c90a198e311154d27304bb09fd06905#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R744) we could not use this because of missing JS features. Has this been resolved by now?

Reply:
Not needed anymore. We only kept the old version because of IE11.